<launch>
  <node pkg="obst_avoid" name="detect_objects_class_test" type="detect_objects_class_test" output="screen"/>
  <node pkg="obst_avoid" name="culculating_opticalflow_class_test" type="culculating_opticalflow_class_test" />
</launch>

